src/test/java/org/broadinstitute/consent/http/resources/DarCollectionResourceTest.java

```diff
@@ -278,6 +278,23 @@ void testGetCollectionWithAllElectionsByCollectionId() {
     assertEquals(HttpStatusCodes.STATUS_CODE_OK, response.getStatus());
   }
 
+  @Test
+  void testGetCollectionWithAllElectionsByCollectionIdDacMember() {
+    DarCollection collection = mockDarCollection();
+    List<Integer> userDatasetIds = List.of(1, 2);
+    User admin = new User(2, authUser.getEmail(), "Dac Member User", new Date(),
+        List.of(UserRoles.Member()));
+    DuosUser duosAdmin = new DuosUser(authUser, admin);
+    collection.setCreateUser(researcher);
+    collection.setCreateUserId(researcher.getUserId());
+
+    when(darCollectionService.findDatasetIdsByDACUser(any())).thenReturn(userDatasetIds);
```

Comment:
As an aside, I've started to see where I can remove the any() matchers in favor of more specific ones (e.g. eq(admin)).